resources/definitions/makeit_pro_m

```diff
@@ -0,0 +1,126 @@
+{
+    "id": "makeit_pro_m",
+    "version": 2,
+    "name": "MAKEiT_ProM",
```

Comment:
Similar to the previous, this string is user-visible and should be following conventional spelling.